CHANGES.rst

```diff
@@ -486,7 +486,7 @@ Bug Fixes
 Other Changes and Additions
 ^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
-- Nothing changed yet.
+- Added instructions for installing Astropy into CASA.
```

Comment:
My sense is that here it is useful: someone who is using astropy but not within CASA may look at what changed may now realise this is possible; very unlikely they would find it in the documentation if they had already concluded it was "hard".